content/community/sig/imaging/index.md

```diff
@@ -0,0 +1,124 @@
+> *Include I2K Poster https://zenodo.org/records/13847465 *
+
+## Who We Are
+
+We are a diverse group of users, developers, trainers, and imaging data enthusiasts, collaborating to expand Galaxy’s image analysis capabilities, bringing the latest tools, feedback, and beyond.
+
+- **Developers**: Are you building image analysis tools in Galaxy, or thinking of starting a project here? Do you need users, testers, or just want to make sure you’re not duplicating tools that already exist? Join us to connect, share ideas, and get support for your development journey!
+
+- **Users**: Do you use Galaxy’s image analysis tools for bio-imaging, Earth observation, or other fields? Have feedback for us, or interested in testing new resources? Perhaps you have ideas or requests for new tools—your input can help shape future developments!
+
+- **Trainers**: Are you using Galaxy to train others in image analysis? If you have suggestions or specific needs to better support your trainees, we’d love to hear from you. Your insights are invaluable in helping us create better learning experiences and share data image analysis knowledge. 
+
+# Getting Started: GTN Tutorials
+
+To dive into Galaxy, the [Galaxy Training Network (GTN)](https://training.galaxyproject.org) offers a wealth of tutorials for beginners and experts alike. These tutorials cover basic and advanced workflows, guiding you through analyses step-by-step. They also include a curated selection of workflows to help you gain hands-on experience in Galaxy.
+
+> *Embed the GTN tutorial table*
+
+# What is available?
+
+## Image Analysis Tools
+
+The Galaxy ToolShed offers a variety of tools for image data analysis, freely available for installation on any Galaxy server. Our community maintains a curated list of available tools to keep users informed and up-to-date.
+
+<iframe
+  id="inlineFrameExample"
+  title="Image analysis related tools"
+  width="100%"
+  height="600"
+  frameBorder="0"
+  src="https://galaxyproject.github.io/galaxy_codex/imaging/">
+</iframe>
+
+> *Is something missing? Reach out to us to keep the list comprehensive.*
+
+## Interactive Environments
+
+Image analysis often involves hands-on interaction, and Galaxy offers resources that make this possible. Check out our interactive tools to experience Galaxy’s potential beyond traditional workflows.
+
+## Visualizations
+
+The **zarr** format is becoming a standard for storing large image data. With various adaptations like OME-Zarr and GEO-Zarr, this interoperable format enhances data reusability across disciplines. Our community supports zarr-based visualization through tools like vizarr, helping bring this dream of cross-discipline interoperability to life.
+
+> *Include vizarr visualization here.*
+
+## A Dedicated Galaxy Interface
+
+Our dedicated interface for image data analysis—[imaging.usegalaxy.eu](https://imaging.usegalaxy.eu)—provides a comprehensive suite of tools and workflows tailored specifically for imaging scientists.
```

Comment:
Maybe I'm missing something here, but how is imaging.usegalaxy.eu a user interface? It's more like an entry page, isn't it?